ocs_ci/deployment/helpers/ceph_cluster.py

```
@@ -0,0 +1,453 @@
+            will be installed. If None, the first worker node in the list will be used as the host node.
+        namespace (str): Namespace for the debug pod.
+
+    Returns:
+        bool: True if the installation succeeded on all nodes, False otherwise.
+
+    Raises:
+        RuntimeError: If the Ceph cluster or configuration installation fails on any node.
+
+    """
+    host_node = host_node or wnodes[0]
+    logger.info(f"Installing minimal Ceph cluster on host node: {host_node.name}")
```

Ruff (0.15.12)

[warning] Logging statement uses f-string

(G004)